src/Components/Authentication/Login.jsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import { Navbar } from '..'
+import "./Login.css"
+import {Link} from "react-router-dom"
```

Comment:
Following one of `{ Navbar }` and `{Navbar}` makes all your imports look uniform